packages/cli/lib/services/dryrun.service.ts

```diff
@@ -604,11 +603,21 @@ export class DryRunService {
                     }
                 } else {
                     const tmp = serializeError(!err || typeof err !== 'object' ? new Error(JSON.stringify(err)) : err);
+
+                    const stacktrace = tmp.stack
+                        ? tmp.stack
+                              .split('\n')
+                              .filter((s, i) => i === 0 || s.includes(filename))
+                              .map((s) => s.trim())
+                              .slice(0, 5) // max 5 lines
+                        : [];
+
                     return {
                         success: false,
                         error: {
                             type: 'script_internal_error',
                             payload: { name: tmp.name || 'Error', code: tmp.code, message: tmp.message },
+                            ...(stacktrace.length > 0 ? { stacktrace } : {}),
```

Comment:
why doesn't it need to be outside of the payload in dryrun?

Reply:
Just to make it more obvious and visible to the user. Plus it reads a bit weird inside because the error is repeated with the payload so imo it’s a bit cleaner to be outside